2021-02-15 12:10:09 Error: [Error] Call to a member function success() on null
Request URL: /acl/groups/add
Stack Trace:
#0 [internal function]: GroupsController->add()
#1 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(GroupsController), Array)
#2 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(GroupsController), Object(CakeRequest))
#4 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-02-15 12:10:20 Error: [Error] Call to a member function success() on null
Request URL: /acl/groups/add
Stack Trace:
#0 [internal function]: GroupsController->add()
#1 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(GroupsController), Array)
#2 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(GroupsController), Object(CakeRequest))
#4 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-02-15 12:10:29 Error: [Error] Call to a member function success() on null
Request URL: /acl/groups/add
Stack Trace:
#0 [internal function]: GroupsController->add()
#1 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(GroupsController), Array)
#2 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(GroupsController), Object(CakeRequest))
#4 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-02-15 12:10:48 Error: [Error] Call to a member function success() on null
Request URL: /acl/users/add
Stack Trace:
#0 [internal function]: UsersController->add()
#1 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#2 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-02-15 12:10:55 Error: [Error] Call to a member function success() on null
Request URL: /acl/users/add
Stack Trace:
#0 [internal function]: UsersController->add()
#1 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#2 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-02-15 12:11:03 Error: [Error] Call to a member function success() on null
Request URL: /acl/users/add
Stack Trace:
#0 [internal function]: UsersController->add()
#1 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#2 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-02-15 12:16:33 Error: [Error] Call to a member function success() on null
Request URL: /acl/groups/add
Stack Trace:
#0 [internal function]: GroupsController->add()
#1 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(GroupsController), Array)
#2 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(GroupsController), Object(CakeRequest))
#4 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-02-15 12:16:38 Error: [Error] Call to a member function success() on null
Request URL: /acl/groups/add
Stack Trace:
#0 [internal function]: GroupsController->add()
#1 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(GroupsController), Array)
#2 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(GroupsController), Object(CakeRequest))
#4 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-02-15 12:17:01 Error: [Error] Call to a member function success() on null
Request URL: /acl/groups/add
Stack Trace:
#0 [internal function]: GroupsController->add()
#1 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(GroupsController), Array)
#2 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(GroupsController), Object(CakeRequest))
#4 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-02-15 12:17:59 Error: [Error] Call to a member function success() on null
Request URL: /acl/users/add
Stack Trace:
#0 [internal function]: UsersController->add()
#1 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#2 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-02-15 12:18:06 Error: [Error] Call to a member function success() on null
Request URL: /acl/users/add
Stack Trace:
#0 [internal function]: UsersController->add()
#1 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#2 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-02-15 12:18:12 Error: [Error] Call to a member function success() on null
Request URL: /acl/users/add
Stack Trace:
#0 [internal function]: UsersController->add()
#1 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#2 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-02-15 12:29:25 Error: [CakeException] AclNode::node() - Couldn't find Aro node identified by "Array
(
    [Aro0.model] => Group
    [Aro0.foreign_key] => 1
)
"
Request URL: /acl/users/initdb
Stack Trace:
#0 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(222): AclNode->node(Array)
#1 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(174): Permission->getAclLink(Object(Group), 'controllers')
#2 C:\wamp64\www\acl\lib\Cake\Controller\Component\Acl\DbAcl.php(87): Permission->allow(Object(Group), 'controllers', '*', 1)
#3 C:\wamp64\www\acl\lib\Cake\Controller\Component\AclComponent.php(123): DbAcl->allow(Object(Group), 'controllers', '*')
#4 C:\wamp64\www\acl\app\Controller\UsersController.php(139): AclComponent->allow(Object(Group), 'controllers')
#5 [internal function]: UsersController->initDB()
#6 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#7 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#8 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#9 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#10 {main}
2021-02-15 12:33:34 Error: [CakeException] AclNode::node() - Couldn't find Aro node identified by "Array
(
    [Aro0.model] => Group
    [Aro0.foreign_key] => 1
)
"
Request URL: /acl/users/initDB
Stack Trace:
#0 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(222): AclNode->node(Array)
#1 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(174): Permission->getAclLink(Object(Group), 'controllers')
#2 C:\wamp64\www\acl\lib\Cake\Controller\Component\Acl\DbAcl.php(87): Permission->allow(Object(Group), 'controllers', '*', 1)
#3 C:\wamp64\www\acl\lib\Cake\Controller\Component\AclComponent.php(123): DbAcl->allow(Object(Group), 'controllers', '*')
#4 C:\wamp64\www\acl\app\Controller\UsersController.php(139): AclComponent->allow(Object(Group), 'controllers')
#5 [internal function]: UsersController->initDB()
#6 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#7 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#8 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#9 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#10 {main}
2021-02-15 12:55:41 Error: [CakeException] AclNode::node() - Couldn't find Aro node identified by "Array
(
    [Aro0.model] => Group
    [Aro0.foreign_key] => 4
)
"
Request URL: /acl/users/add
Stack Trace:
#0 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(80): AclNode->node(Array)
#1 C:\wamp64\www\acl\lib\Cake\Controller\Component\Acl\DbAcl.php(73): Permission->check(Array, 'controllers/Use...', '*')
#2 C:\wamp64\www\acl\lib\Cake\Controller\Component\AclComponent.php(110): DbAcl->check(Array, 'controllers/Use...', '*')
#3 C:\wamp64\www\acl\lib\Cake\Controller\Component\Auth\ActionsAuthorize.php(38): AclComponent->check(Array, 'controllers/Use...')
#4 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(470): ActionsAuthorize->authorize(Array, Object(CakeRequest))
#5 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(311): AuthComponent->isAuthorized(Array)
#6 C:\wamp64\www\acl\lib\Cake\Utility\ObjectCollection.php(129): AuthComponent->startup(Object(UsersController))
#7 C:\wamp64\www\acl\lib\Cake\Event\CakeEventManager.php(244): ObjectCollection->trigger('startup')
#8 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(683): CakeEventManager->dispatch(Object(CakeEvent))
#9 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(189): Controller->startupProcess()
#10 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#11 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2021-02-15 12:56:18 Error: [CakeException] AclNode::node() - Couldn't find Aro node identified by "Array
(
    [Aro0.model] => Group
    [Aro0.foreign_key] => 4
)
"
Request URL: /acl/posts/add
Stack Trace:
#0 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(80): AclNode->node(Array)
#1 C:\wamp64\www\acl\lib\Cake\Controller\Component\Acl\DbAcl.php(73): Permission->check(Array, 'controllers/Pos...', '*')
#2 C:\wamp64\www\acl\lib\Cake\Controller\Component\AclComponent.php(110): DbAcl->check(Array, 'controllers/Pos...', '*')
#3 C:\wamp64\www\acl\lib\Cake\Controller\Component\Auth\ActionsAuthorize.php(38): AclComponent->check(Array, 'controllers/Pos...')
#4 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(470): ActionsAuthorize->authorize(Array, Object(CakeRequest))
#5 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(311): AuthComponent->isAuthorized(Array)
#6 C:\wamp64\www\acl\lib\Cake\Utility\ObjectCollection.php(129): AuthComponent->startup(Object(PostsController))
#7 C:\wamp64\www\acl\lib\Cake\Event\CakeEventManager.php(244): ObjectCollection->trigger('startup')
#8 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(683): CakeEventManager->dispatch(Object(CakeEvent))
#9 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(189): Controller->startupProcess()
#10 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest))
#11 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2021-02-15 12:56:24 Error: [CakeException] AclNode::node() - Couldn't find Aro node identified by "Array
(
    [Aro0.model] => Group
    [Aro0.foreign_key] => 4
)
"
Request URL: /acl/users
Stack Trace:
#0 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(80): AclNode->node(Array)
#1 C:\wamp64\www\acl\lib\Cake\Controller\Component\Acl\DbAcl.php(73): Permission->check(Array, 'controllers/Use...', '*')
#2 C:\wamp64\www\acl\lib\Cake\Controller\Component\AclComponent.php(110): DbAcl->check(Array, 'controllers/Use...', '*')
#3 C:\wamp64\www\acl\lib\Cake\Controller\Component\Auth\ActionsAuthorize.php(38): AclComponent->check(Array, 'controllers/Use...')
#4 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(470): ActionsAuthorize->authorize(Array, Object(CakeRequest))
#5 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(311): AuthComponent->isAuthorized(Array)
#6 C:\wamp64\www\acl\lib\Cake\Utility\ObjectCollection.php(129): AuthComponent->startup(Object(UsersController))
#7 C:\wamp64\www\acl\lib\Cake\Event\CakeEventManager.php(244): ObjectCollection->trigger('startup')
#8 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(683): CakeEventManager->dispatch(Object(CakeEvent))
#9 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(189): Controller->startupProcess()
#10 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#11 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2021-02-15 12:56:36 Error: [CakeException] AclNode::node() - Couldn't find Aro node identified by "Array
(
    [Aro0.model] => Group
    [Aro0.foreign_key] => 4
)
"
Request URL: /acl/users/add
Stack Trace:
#0 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(80): AclNode->node(Array)
#1 C:\wamp64\www\acl\lib\Cake\Controller\Component\Acl\DbAcl.php(73): Permission->check(Array, 'controllers/Use...', '*')
#2 C:\wamp64\www\acl\lib\Cake\Controller\Component\AclComponent.php(110): DbAcl->check(Array, 'controllers/Use...', '*')
#3 C:\wamp64\www\acl\lib\Cake\Controller\Component\Auth\ActionsAuthorize.php(38): AclComponent->check(Array, 'controllers/Use...')
#4 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(470): ActionsAuthorize->authorize(Array, Object(CakeRequest))
#5 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(311): AuthComponent->isAuthorized(Array)
#6 C:\wamp64\www\acl\lib\Cake\Utility\ObjectCollection.php(129): AuthComponent->startup(Object(UsersController))
#7 C:\wamp64\www\acl\lib\Cake\Event\CakeEventManager.php(244): ObjectCollection->trigger('startup')
#8 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(683): CakeEventManager->dispatch(Object(CakeEvent))
#9 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(189): Controller->startupProcess()
#10 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#11 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2021-02-15 12:56:44 Error: [CakeException] AclNode::node() - Couldn't find Aro node identified by "Array
(
    [Aro0.model] => Group
    [Aro0.foreign_key] => 4
)
"
Request URL: /acl/widgets/add
Stack Trace:
#0 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(80): AclNode->node(Array)
#1 C:\wamp64\www\acl\lib\Cake\Controller\Component\Acl\DbAcl.php(73): Permission->check(Array, 'controllers/Wid...', '*')
#2 C:\wamp64\www\acl\lib\Cake\Controller\Component\AclComponent.php(110): DbAcl->check(Array, 'controllers/Wid...', '*')
#3 C:\wamp64\www\acl\lib\Cake\Controller\Component\Auth\ActionsAuthorize.php(38): AclComponent->check(Array, 'controllers/Wid...')
#4 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(470): ActionsAuthorize->authorize(Array, Object(CakeRequest))
#5 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(311): AuthComponent->isAuthorized(Array)
#6 C:\wamp64\www\acl\lib\Cake\Utility\ObjectCollection.php(129): AuthComponent->startup(Object(WidgetsController))
#7 C:\wamp64\www\acl\lib\Cake\Event\CakeEventManager.php(244): ObjectCollection->trigger('startup')
#8 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(683): CakeEventManager->dispatch(Object(CakeEvent))
#9 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(189): Controller->startupProcess()
#10 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(WidgetsController), Object(CakeRequest))
#11 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2021-02-15 12:56:54 Error: [CakeException] AclNode::node() - Couldn't find Aro node identified by "Array
(
    [Aro0.model] => Group
    [Aro0.foreign_key] => 4
)
"
Request URL: /acl/users/logout
Stack Trace:
#0 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(80): AclNode->node(Array)
#1 C:\wamp64\www\acl\lib\Cake\Controller\Component\Acl\DbAcl.php(73): Permission->check(Array, 'controllers/Use...', '*')
#2 C:\wamp64\www\acl\lib\Cake\Controller\Component\AclComponent.php(110): DbAcl->check(Array, 'controllers/Use...', '*')
#3 C:\wamp64\www\acl\lib\Cake\Controller\Component\Auth\ActionsAuthorize.php(38): AclComponent->check(Array, 'controllers/Use...')
#4 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(470): ActionsAuthorize->authorize(Array, Object(CakeRequest))
#5 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(311): AuthComponent->isAuthorized(Array)
#6 C:\wamp64\www\acl\lib\Cake\Utility\ObjectCollection.php(129): AuthComponent->startup(Object(UsersController))
#7 C:\wamp64\www\acl\lib\Cake\Event\CakeEventManager.php(244): ObjectCollection->trigger('startup')
#8 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(683): CakeEventManager->dispatch(Object(CakeEvent))
#9 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(189): Controller->startupProcess()
#10 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#11 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2021-02-15 12:57:47 Error: [CakeException] AclNode::node() - Couldn't find Aro node identified by "Array
(
    [Aro0.model] => Group
    [Aro0.foreign_key] => 4
)
"
Request URL: /acl/users/logout
Stack Trace:
#0 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(80): AclNode->node(Array)
#1 C:\wamp64\www\acl\lib\Cake\Controller\Component\Acl\DbAcl.php(73): Permission->check(Array, 'controllers/Use...', '*')
#2 C:\wamp64\www\acl\lib\Cake\Controller\Component\AclComponent.php(110): DbAcl->check(Array, 'controllers/Use...', '*')
#3 C:\wamp64\www\acl\lib\Cake\Controller\Component\Auth\ActionsAuthorize.php(38): AclComponent->check(Array, 'controllers/Use...')
#4 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(470): ActionsAuthorize->authorize(Array, Object(CakeRequest))
#5 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(311): AuthComponent->isAuthorized(Array)
#6 C:\wamp64\www\acl\lib\Cake\Utility\ObjectCollection.php(129): AuthComponent->startup(Object(UsersController))
#7 C:\wamp64\www\acl\lib\Cake\Event\CakeEventManager.php(244): ObjectCollection->trigger('startup')
#8 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(683): CakeEventManager->dispatch(Object(CakeEvent))
#9 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(189): Controller->startupProcess()
#10 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#11 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2021-02-15 12:57:58 Error: [CakeException] AclNode::node() - Couldn't find Aro node identified by "Array
(
    [Aro0.model] => Group
    [Aro0.foreign_key] => 4
)
"
Request URL: /acl/users
Stack Trace:
#0 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(80): AclNode->node(Array)
#1 C:\wamp64\www\acl\lib\Cake\Controller\Component\Acl\DbAcl.php(73): Permission->check(Array, 'controllers/Use...', '*')
#2 C:\wamp64\www\acl\lib\Cake\Controller\Component\AclComponent.php(110): DbAcl->check(Array, 'controllers/Use...', '*')
#3 C:\wamp64\www\acl\lib\Cake\Controller\Component\Auth\ActionsAuthorize.php(38): AclComponent->check(Array, 'controllers/Use...')
#4 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(470): ActionsAuthorize->authorize(Array, Object(CakeRequest))
#5 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(311): AuthComponent->isAuthorized(Array)
#6 C:\wamp64\www\acl\lib\Cake\Utility\ObjectCollection.php(129): AuthComponent->startup(Object(UsersController))
#7 C:\wamp64\www\acl\lib\Cake\Event\CakeEventManager.php(244): ObjectCollection->trigger('startup')
#8 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(683): CakeEventManager->dispatch(Object(CakeEvent))
#9 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(189): Controller->startupProcess()
#10 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#11 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2021-02-15 12:58:04 Error: [CakeException] AclNode::node() - Couldn't find Aro node identified by "Array
(
    [Aro0.model] => Group
    [Aro0.foreign_key] => 4
)
"
Request URL: /acl/users
Stack Trace:
#0 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(80): AclNode->node(Array)
#1 C:\wamp64\www\acl\lib\Cake\Controller\Component\Acl\DbAcl.php(73): Permission->check(Array, 'controllers/Use...', '*')
#2 C:\wamp64\www\acl\lib\Cake\Controller\Component\AclComponent.php(110): DbAcl->check(Array, 'controllers/Use...', '*')
#3 C:\wamp64\www\acl\lib\Cake\Controller\Component\Auth\ActionsAuthorize.php(38): AclComponent->check(Array, 'controllers/Use...')
#4 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(470): ActionsAuthorize->authorize(Array, Object(CakeRequest))
#5 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(311): AuthComponent->isAuthorized(Array)
#6 C:\wamp64\www\acl\lib\Cake\Utility\ObjectCollection.php(129): AuthComponent->startup(Object(UsersController))
#7 C:\wamp64\www\acl\lib\Cake\Event\CakeEventManager.php(244): ObjectCollection->trigger('startup')
#8 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(683): CakeEventManager->dispatch(Object(CakeEvent))
#9 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(189): Controller->startupProcess()
#10 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#11 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2021-02-15 12:58:42 Error: [CakeException] AclNode::node() - Couldn't find Aro node identified by "Array
(
    [Aro0.model] => Group
    [Aro0.foreign_key] => 4
)
"
Request URL: /acl/users
Stack Trace:
#0 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(80): AclNode->node(Array)
#1 C:\wamp64\www\acl\lib\Cake\Controller\Component\Acl\DbAcl.php(73): Permission->check(Array, 'controllers/Use...', '*')
#2 C:\wamp64\www\acl\lib\Cake\Controller\Component\AclComponent.php(110): DbAcl->check(Array, 'controllers/Use...', '*')
#3 C:\wamp64\www\acl\lib\Cake\Controller\Component\Auth\ActionsAuthorize.php(38): AclComponent->check(Array, 'controllers/Use...')
#4 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(470): ActionsAuthorize->authorize(Array, Object(CakeRequest))
#5 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(311): AuthComponent->isAuthorized(Array)
#6 C:\wamp64\www\acl\lib\Cake\Utility\ObjectCollection.php(129): AuthComponent->startup(Object(UsersController))
#7 C:\wamp64\www\acl\lib\Cake\Event\CakeEventManager.php(244): ObjectCollection->trigger('startup')
#8 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(683): CakeEventManager->dispatch(Object(CakeEvent))
#9 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(189): Controller->startupProcess()
#10 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#11 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2021-02-15 12:58:52 Error: [CakeException] AclNode::node() - Couldn't find Aro node identified by "Array
(
    [Aro0.model] => Group
    [Aro0.foreign_key] => 4
)
"
Request URL: /acl/users
Stack Trace:
#0 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(80): AclNode->node(Array)
#1 C:\wamp64\www\acl\lib\Cake\Controller\Component\Acl\DbAcl.php(73): Permission->check(Array, 'controllers/Use...', '*')
#2 C:\wamp64\www\acl\lib\Cake\Controller\Component\AclComponent.php(110): DbAcl->check(Array, 'controllers/Use...', '*')
#3 C:\wamp64\www\acl\lib\Cake\Controller\Component\Auth\ActionsAuthorize.php(38): AclComponent->check(Array, 'controllers/Use...')
#4 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(470): ActionsAuthorize->authorize(Array, Object(CakeRequest))
#5 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(311): AuthComponent->isAuthorized(Array)
#6 C:\wamp64\www\acl\lib\Cake\Utility\ObjectCollection.php(129): AuthComponent->startup(Object(UsersController))
#7 C:\wamp64\www\acl\lib\Cake\Event\CakeEventManager.php(244): ObjectCollection->trigger('startup')
#8 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(683): CakeEventManager->dispatch(Object(CakeEvent))
#9 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(189): Controller->startupProcess()
#10 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#11 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2021-02-15 13:00:46 Error: [CakeException] AclNode::node() - Couldn't find Aro node identified by "Array
(
    [Aro0.model] => Group
    [Aro0.foreign_key] => 4
)
"
Request URL: /acl/users
Stack Trace:
#0 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(80): AclNode->node(Array)
#1 C:\wamp64\www\acl\lib\Cake\Controller\Component\Acl\DbAcl.php(73): Permission->check(Array, 'controllers/Use...', '*')
#2 C:\wamp64\www\acl\lib\Cake\Controller\Component\AclComponent.php(110): DbAcl->check(Array, 'controllers/Use...', '*')
#3 C:\wamp64\www\acl\lib\Cake\Controller\Component\Auth\ActionsAuthorize.php(38): AclComponent->check(Array, 'controllers/Use...')
#4 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(470): ActionsAuthorize->authorize(Array, Object(CakeRequest))
#5 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(311): AuthComponent->isAuthorized(Array)
#6 C:\wamp64\www\acl\lib\Cake\Utility\ObjectCollection.php(129): AuthComponent->startup(Object(UsersController))
#7 C:\wamp64\www\acl\lib\Cake\Event\CakeEventManager.php(244): ObjectCollection->trigger('startup')
#8 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(683): CakeEventManager->dispatch(Object(CakeEvent))
#9 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(189): Controller->startupProcess()
#10 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#11 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2021-02-15 13:01:02 Error: [CakeException] AclNode::node() - Couldn't find Aro node identified by "Array
(
    [Aro0.model] => Group
    [Aro0.foreign_key] => 4
)
"
Request URL: /acl/users
Stack Trace:
#0 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(80): AclNode->node(Array)
#1 C:\wamp64\www\acl\lib\Cake\Controller\Component\Acl\DbAcl.php(73): Permission->check(Array, 'controllers/Use...', '*')
#2 C:\wamp64\www\acl\lib\Cake\Controller\Component\AclComponent.php(110): DbAcl->check(Array, 'controllers/Use...', '*')
#3 C:\wamp64\www\acl\lib\Cake\Controller\Component\Auth\ActionsAuthorize.php(38): AclComponent->check(Array, 'controllers/Use...')
#4 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(470): ActionsAuthorize->authorize(Array, Object(CakeRequest))
#5 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(311): AuthComponent->isAuthorized(Array)
#6 C:\wamp64\www\acl\lib\Cake\Utility\ObjectCollection.php(129): AuthComponent->startup(Object(UsersController))
#7 C:\wamp64\www\acl\lib\Cake\Event\CakeEventManager.php(244): ObjectCollection->trigger('startup')
#8 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(683): CakeEventManager->dispatch(Object(CakeEvent))
#9 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(189): Controller->startupProcess()
#10 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#11 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2021-02-15 13:01:40 Error: [CakeException] AclNode::node() - Couldn't find Aro node identified by "Array
(
    [Aro0.model] => Group
    [Aro0.foreign_key] => 4
)
"
Request URL: /acl/users
Stack Trace:
#0 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(80): AclNode->node(Array)
#1 C:\wamp64\www\acl\lib\Cake\Controller\Component\Acl\DbAcl.php(73): Permission->check(Array, 'controllers/Use...', '*')
#2 C:\wamp64\www\acl\lib\Cake\Controller\Component\AclComponent.php(110): DbAcl->check(Array, 'controllers/Use...', '*')
#3 C:\wamp64\www\acl\lib\Cake\Controller\Component\Auth\ActionsAuthorize.php(38): AclComponent->check(Array, 'controllers/Use...')
#4 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(470): ActionsAuthorize->authorize(Array, Object(CakeRequest))
#5 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(311): AuthComponent->isAuthorized(Array)
#6 C:\wamp64\www\acl\lib\Cake\Utility\ObjectCollection.php(129): AuthComponent->startup(Object(UsersController))
#7 C:\wamp64\www\acl\lib\Cake\Event\CakeEventManager.php(244): ObjectCollection->trigger('startup')
#8 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(683): CakeEventManager->dispatch(Object(CakeEvent))
#9 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(189): Controller->startupProcess()
#10 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#11 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2021-02-15 13:01:41 Error: [CakeException] AclNode::node() - Couldn't find Aro node identified by "Array
(
    [Aro0.model] => Group
    [Aro0.foreign_key] => 4
)
"
Request URL: /acl/users
Stack Trace:
#0 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(80): AclNode->node(Array)
#1 C:\wamp64\www\acl\lib\Cake\Controller\Component\Acl\DbAcl.php(73): Permission->check(Array, 'controllers/Use...', '*')
#2 C:\wamp64\www\acl\lib\Cake\Controller\Component\AclComponent.php(110): DbAcl->check(Array, 'controllers/Use...', '*')
#3 C:\wamp64\www\acl\lib\Cake\Controller\Component\Auth\ActionsAuthorize.php(38): AclComponent->check(Array, 'controllers/Use...')
#4 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(470): ActionsAuthorize->authorize(Array, Object(CakeRequest))
#5 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(311): AuthComponent->isAuthorized(Array)
#6 C:\wamp64\www\acl\lib\Cake\Utility\ObjectCollection.php(129): AuthComponent->startup(Object(UsersController))
#7 C:\wamp64\www\acl\lib\Cake\Event\CakeEventManager.php(244): ObjectCollection->trigger('startup')
#8 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(683): CakeEventManager->dispatch(Object(CakeEvent))
#9 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(189): Controller->startupProcess()
#10 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#11 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2021-02-15 13:01:41 Error: [CakeException] AclNode::node() - Couldn't find Aro node identified by "Array
(
    [Aro0.model] => Group
    [Aro0.foreign_key] => 4
)
"
Request URL: /acl/users
Stack Trace:
#0 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(80): AclNode->node(Array)
#1 C:\wamp64\www\acl\lib\Cake\Controller\Component\Acl\DbAcl.php(73): Permission->check(Array, 'controllers/Use...', '*')
#2 C:\wamp64\www\acl\lib\Cake\Controller\Component\AclComponent.php(110): DbAcl->check(Array, 'controllers/Use...', '*')
#3 C:\wamp64\www\acl\lib\Cake\Controller\Component\Auth\ActionsAuthorize.php(38): AclComponent->check(Array, 'controllers/Use...')
#4 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(470): ActionsAuthorize->authorize(Array, Object(CakeRequest))
#5 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(311): AuthComponent->isAuthorized(Array)
#6 C:\wamp64\www\acl\lib\Cake\Utility\ObjectCollection.php(129): AuthComponent->startup(Object(UsersController))
#7 C:\wamp64\www\acl\lib\Cake\Event\CakeEventManager.php(244): ObjectCollection->trigger('startup')
#8 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(683): CakeEventManager->dispatch(Object(CakeEvent))
#9 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(189): Controller->startupProcess()
#10 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#11 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2021-02-15 13:01:41 Error: [CakeException] AclNode::node() - Couldn't find Aro node identified by "Array
(
    [Aro0.model] => Group
    [Aro0.foreign_key] => 4
)
"
Request URL: /acl/users
Stack Trace:
#0 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(80): AclNode->node(Array)
#1 C:\wamp64\www\acl\lib\Cake\Controller\Component\Acl\DbAcl.php(73): Permission->check(Array, 'controllers/Use...', '*')
#2 C:\wamp64\www\acl\lib\Cake\Controller\Component\AclComponent.php(110): DbAcl->check(Array, 'controllers/Use...', '*')
#3 C:\wamp64\www\acl\lib\Cake\Controller\Component\Auth\ActionsAuthorize.php(38): AclComponent->check(Array, 'controllers/Use...')
#4 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(470): ActionsAuthorize->authorize(Array, Object(CakeRequest))
#5 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(311): AuthComponent->isAuthorized(Array)
#6 C:\wamp64\www\acl\lib\Cake\Utility\ObjectCollection.php(129): AuthComponent->startup(Object(UsersController))
#7 C:\wamp64\www\acl\lib\Cake\Event\CakeEventManager.php(244): ObjectCollection->trigger('startup')
#8 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(683): CakeEventManager->dispatch(Object(CakeEvent))
#9 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(189): Controller->startupProcess()
#10 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#11 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2021-02-15 13:01:46 Error: [CakeException] AclNode::node() - Couldn't find Aro node identified by "Array
(
    [Aro0.model] => Group
    [Aro0.foreign_key] => 4
)
"
Request URL: /acl/users/add
Stack Trace:
#0 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(80): AclNode->node(Array)
#1 C:\wamp64\www\acl\lib\Cake\Controller\Component\Acl\DbAcl.php(73): Permission->check(Array, 'controllers/Use...', '*')
#2 C:\wamp64\www\acl\lib\Cake\Controller\Component\AclComponent.php(110): DbAcl->check(Array, 'controllers/Use...', '*')
#3 C:\wamp64\www\acl\lib\Cake\Controller\Component\Auth\ActionsAuthorize.php(38): AclComponent->check(Array, 'controllers/Use...')
#4 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(470): ActionsAuthorize->authorize(Array, Object(CakeRequest))
#5 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(311): AuthComponent->isAuthorized(Array)
#6 C:\wamp64\www\acl\lib\Cake\Utility\ObjectCollection.php(129): AuthComponent->startup(Object(UsersController))
#7 C:\wamp64\www\acl\lib\Cake\Event\CakeEventManager.php(244): ObjectCollection->trigger('startup')
#8 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(683): CakeEventManager->dispatch(Object(CakeEvent))
#9 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(189): Controller->startupProcess()
#10 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#11 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2021-02-15 13:05:41 Error: [CakeException] AclNode::node() - Couldn't find Aro node identified by "Array
(
    [Aro0.model] => Group
    [Aro0.foreign_key] => 4
)
"
Request URL: /acl/users/logout
Stack Trace:
#0 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(80): AclNode->node(Array)
#1 C:\wamp64\www\acl\lib\Cake\Controller\Component\Acl\DbAcl.php(73): Permission->check(Array, 'controllers/Use...', '*')
#2 C:\wamp64\www\acl\lib\Cake\Controller\Component\AclComponent.php(110): DbAcl->check(Array, 'controllers/Use...', '*')
#3 C:\wamp64\www\acl\lib\Cake\Controller\Component\Auth\ActionsAuthorize.php(38): AclComponent->check(Array, 'controllers/Use...')
#4 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(470): ActionsAuthorize->authorize(Array, Object(CakeRequest))
#5 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(311): AuthComponent->isAuthorized(Array)
#6 C:\wamp64\www\acl\lib\Cake\Utility\ObjectCollection.php(129): AuthComponent->startup(Object(UsersController))
#7 C:\wamp64\www\acl\lib\Cake\Event\CakeEventManager.php(244): ObjectCollection->trigger('startup')
#8 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(683): CakeEventManager->dispatch(Object(CakeEvent))
#9 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(189): Controller->startupProcess()
#10 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#11 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2021-02-15 13:06:49 Error: [CakeException] AclNode::node() - Couldn't find Aro node identified by "Array
(
    [Aro0.model] => Group
    [Aro0.foreign_key] => 4
)
"
Request URL: /acl/users/add
Stack Trace:
#0 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(80): AclNode->node(Array)
#1 C:\wamp64\www\acl\lib\Cake\Controller\Component\Acl\DbAcl.php(73): Permission->check(Array, 'controllers/Use...', '*')
#2 C:\wamp64\www\acl\lib\Cake\Controller\Component\AclComponent.php(110): DbAcl->check(Array, 'controllers/Use...', '*')
#3 C:\wamp64\www\acl\lib\Cake\Controller\Component\Auth\ActionsAuthorize.php(38): AclComponent->check(Array, 'controllers/Use...')
#4 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(470): ActionsAuthorize->authorize(Array, Object(CakeRequest))
#5 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(311): AuthComponent->isAuthorized(Array)
#6 C:\wamp64\www\acl\lib\Cake\Utility\ObjectCollection.php(129): AuthComponent->startup(Object(UsersController))
#7 C:\wamp64\www\acl\lib\Cake\Event\CakeEventManager.php(244): ObjectCollection->trigger('startup')
#8 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(683): CakeEventManager->dispatch(Object(CakeEvent))
#9 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(189): Controller->startupProcess()
#10 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#11 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2021-02-15 13:07:35 Error: [CakeException] AclNode::node() - Couldn't find Aro node identified by "Array
(
    [Aro0.model] => Group
    [Aro0.foreign_key] => 4
)
"
Request URL: /acl/users/add
Stack Trace:
#0 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(80): AclNode->node(Array)
#1 C:\wamp64\www\acl\lib\Cake\Controller\Component\Acl\DbAcl.php(73): Permission->check(Array, 'controllers/Use...', '*')
#2 C:\wamp64\www\acl\lib\Cake\Controller\Component\AclComponent.php(110): DbAcl->check(Array, 'controllers/Use...', '*')
#3 C:\wamp64\www\acl\lib\Cake\Controller\Component\Auth\ActionsAuthorize.php(38): AclComponent->check(Array, 'controllers/Use...')
#4 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(470): ActionsAuthorize->authorize(Array, Object(CakeRequest))
#5 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(311): AuthComponent->isAuthorized(Array)
#6 C:\wamp64\www\acl\lib\Cake\Utility\ObjectCollection.php(129): AuthComponent->startup(Object(UsersController))
#7 C:\wamp64\www\acl\lib\Cake\Event\CakeEventManager.php(244): ObjectCollection->trigger('startup')
#8 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(683): CakeEventManager->dispatch(Object(CakeEvent))
#9 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(189): Controller->startupProcess()
#10 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#11 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2021-02-15 13:08:18 Error: [CakeException] AclNode::node() - Couldn't find Aro node identified by "Array
(
    [Aro0.model] => Group
    [Aro0.foreign_key] => 4
)
"
Request URL: /acl/posts/add
Stack Trace:
#0 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(80): AclNode->node(Array)
#1 C:\wamp64\www\acl\lib\Cake\Controller\Component\Acl\DbAcl.php(73): Permission->check(Array, 'controllers/Pos...', '*')
#2 C:\wamp64\www\acl\lib\Cake\Controller\Component\AclComponent.php(110): DbAcl->check(Array, 'controllers/Pos...', '*')
#3 C:\wamp64\www\acl\lib\Cake\Controller\Component\Auth\ActionsAuthorize.php(38): AclComponent->check(Array, 'controllers/Pos...')
#4 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(470): ActionsAuthorize->authorize(Array, Object(CakeRequest))
#5 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(311): AuthComponent->isAuthorized(Array)
#6 C:\wamp64\www\acl\lib\Cake\Utility\ObjectCollection.php(129): AuthComponent->startup(Object(PostsController))
#7 C:\wamp64\www\acl\lib\Cake\Event\CakeEventManager.php(244): ObjectCollection->trigger('startup')
#8 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(683): CakeEventManager->dispatch(Object(CakeEvent))
#9 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(189): Controller->startupProcess()
#10 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest))
#11 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2021-02-15 13:09:50 Error: [CakeException] AclNode::node() - Couldn't find Aro node identified by "Array
(
    [Aro0.model] => Group
    [Aro0.foreign_key] => 4
)
"
Request URL: /acl/posts/add
Stack Trace:
#0 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(80): AclNode->node(Array)
#1 C:\wamp64\www\acl\lib\Cake\Controller\Component\Acl\DbAcl.php(73): Permission->check(Array, 'controllers/Pos...', '*')
#2 C:\wamp64\www\acl\lib\Cake\Controller\Component\AclComponent.php(110): DbAcl->check(Array, 'controllers/Pos...', '*')
#3 C:\wamp64\www\acl\lib\Cake\Controller\Component\Auth\ActionsAuthorize.php(38): AclComponent->check(Array, 'controllers/Pos...')
#4 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(470): ActionsAuthorize->authorize(Array, Object(CakeRequest))
#5 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(311): AuthComponent->isAuthorized(Array)
#6 C:\wamp64\www\acl\lib\Cake\Utility\ObjectCollection.php(129): AuthComponent->startup(Object(PostsController))
#7 C:\wamp64\www\acl\lib\Cake\Event\CakeEventManager.php(244): ObjectCollection->trigger('startup')
#8 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(683): CakeEventManager->dispatch(Object(CakeEvent))
#9 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(189): Controller->startupProcess()
#10 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest))
#11 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2021-02-15 13:10:03 Error: [CakeException] AclNode::node() - Couldn't find Aro node identified by "Array
(
    [Aro0.model] => Group
    [Aro0.foreign_key] => 4
)
"
Request URL: /acl/posts/add
Stack Trace:
#0 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(80): AclNode->node(Array)
#1 C:\wamp64\www\acl\lib\Cake\Controller\Component\Acl\DbAcl.php(73): Permission->check(Array, 'controllers/Pos...', '*')
#2 C:\wamp64\www\acl\lib\Cake\Controller\Component\AclComponent.php(110): DbAcl->check(Array, 'controllers/Pos...', '*')
#3 C:\wamp64\www\acl\lib\Cake\Controller\Component\Auth\ActionsAuthorize.php(38): AclComponent->check(Array, 'controllers/Pos...')
#4 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(470): ActionsAuthorize->authorize(Array, Object(CakeRequest))
#5 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(311): AuthComponent->isAuthorized(Array)
#6 C:\wamp64\www\acl\lib\Cake\Utility\ObjectCollection.php(129): AuthComponent->startup(Object(PostsController))
#7 C:\wamp64\www\acl\lib\Cake\Event\CakeEventManager.php(244): ObjectCollection->trigger('startup')
#8 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(683): CakeEventManager->dispatch(Object(CakeEvent))
#9 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(189): Controller->startupProcess()
#10 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest))
#11 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2021-02-15 13:18:18 Error: [CakeException] AclNode::node() - Couldn't find Aro node identified by "Array
(
    [Aro0.model] => Group
    [Aro0.foreign_key] => 4
)
"
Request URL: /acl/posts/add
Stack Trace:
#0 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(80): AclNode->node(Array)
#1 C:\wamp64\www\acl\lib\Cake\Controller\Component\Acl\DbAcl.php(73): Permission->check(Array, 'controllers/Pos...', '*')
#2 C:\wamp64\www\acl\lib\Cake\Controller\Component\AclComponent.php(110): DbAcl->check(Array, 'controllers/Pos...', '*')
#3 C:\wamp64\www\acl\lib\Cake\Controller\Component\Auth\ActionsAuthorize.php(38): AclComponent->check(Array, 'controllers/Pos...')
#4 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(470): ActionsAuthorize->authorize(Array, Object(CakeRequest))
#5 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(311): AuthComponent->isAuthorized(Array)
#6 C:\wamp64\www\acl\lib\Cake\Utility\ObjectCollection.php(129): AuthComponent->startup(Object(PostsController))
#7 C:\wamp64\www\acl\lib\Cake\Event\CakeEventManager.php(244): ObjectCollection->trigger('startup')
#8 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(683): CakeEventManager->dispatch(Object(CakeEvent))
#9 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(189): Controller->startupProcess()
#10 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest))
#11 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2021-02-15 13:18:25 Error: [CakeException] AclNode::node() - Couldn't find Aro node identified by "Array
(
    [Aro0.model] => Group
    [Aro0.foreign_key] => 4
)
"
Request URL: /acl/users
Stack Trace:
#0 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(80): AclNode->node(Array)
#1 C:\wamp64\www\acl\lib\Cake\Controller\Component\Acl\DbAcl.php(73): Permission->check(Array, 'controllers/Use...', '*')
#2 C:\wamp64\www\acl\lib\Cake\Controller\Component\AclComponent.php(110): DbAcl->check(Array, 'controllers/Use...', '*')
#3 C:\wamp64\www\acl\lib\Cake\Controller\Component\Auth\ActionsAuthorize.php(38): AclComponent->check(Array, 'controllers/Use...')
#4 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(470): ActionsAuthorize->authorize(Array, Object(CakeRequest))
#5 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(311): AuthComponent->isAuthorized(Array)
#6 C:\wamp64\www\acl\lib\Cake\Utility\ObjectCollection.php(129): AuthComponent->startup(Object(UsersController))
#7 C:\wamp64\www\acl\lib\Cake\Event\CakeEventManager.php(244): ObjectCollection->trigger('startup')
#8 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(683): CakeEventManager->dispatch(Object(CakeEvent))
#9 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(189): Controller->startupProcess()
#10 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#11 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2021-02-15 13:18:41 Error: [CakeException] AclNode::node() - Couldn't find Aro node identified by "Array
(
    [Aro0.model] => Group
    [Aro0.foreign_key] => 4
)
"
Request URL: /acl/users/add
Stack Trace:
#0 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(80): AclNode->node(Array)
#1 C:\wamp64\www\acl\lib\Cake\Controller\Component\Acl\DbAcl.php(73): Permission->check(Array, 'controllers/Use...', '*')
#2 C:\wamp64\www\acl\lib\Cake\Controller\Component\AclComponent.php(110): DbAcl->check(Array, 'controllers/Use...', '*')
#3 C:\wamp64\www\acl\lib\Cake\Controller\Component\Auth\ActionsAuthorize.php(38): AclComponent->check(Array, 'controllers/Use...')
#4 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(470): ActionsAuthorize->authorize(Array, Object(CakeRequest))
#5 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(311): AuthComponent->isAuthorized(Array)
#6 C:\wamp64\www\acl\lib\Cake\Utility\ObjectCollection.php(129): AuthComponent->startup(Object(UsersController))
#7 C:\wamp64\www\acl\lib\Cake\Event\CakeEventManager.php(244): ObjectCollection->trigger('startup')
#8 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(683): CakeEventManager->dispatch(Object(CakeEvent))
#9 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(189): Controller->startupProcess()
#10 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#11 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2021-02-15 13:18:54 Error: [CakeException] AclNode::node() - Couldn't find Aro node identified by "Array
(
    [Aro0.model] => Group
    [Aro0.foreign_key] => 4
)
"
Request URL: /acl/users/logout
Stack Trace:
#0 C:\wamp64\www\acl\lib\Cake\Model\Permission.php(80): AclNode->node(Array)
#1 C:\wamp64\www\acl\lib\Cake\Controller\Component\Acl\DbAcl.php(73): Permission->check(Array, 'controllers/Use...', '*')
#2 C:\wamp64\www\acl\lib\Cake\Controller\Component\AclComponent.php(110): DbAcl->check(Array, 'controllers/Use...', '*')
#3 C:\wamp64\www\acl\lib\Cake\Controller\Component\Auth\ActionsAuthorize.php(38): AclComponent->check(Array, 'controllers/Use...')
#4 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(470): ActionsAuthorize->authorize(Array, Object(CakeRequest))
#5 C:\wamp64\www\acl\lib\Cake\Controller\Component\AuthComponent.php(311): AuthComponent->isAuthorized(Array)
#6 C:\wamp64\www\acl\lib\Cake\Utility\ObjectCollection.php(129): AuthComponent->startup(Object(UsersController))
#7 C:\wamp64\www\acl\lib\Cake\Event\CakeEventManager.php(244): ObjectCollection->trigger('startup')
#8 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(683): CakeEventManager->dispatch(Object(CakeEvent))
#9 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(189): Controller->startupProcess()
#10 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#11 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#12 {main}
2021-02-15 13:21:51 Error: [Error] Call to a member function success() on null
Request URL: /acl/users/add
Stack Trace:
#0 [internal function]: UsersController->add()
#1 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#2 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-02-15 13:22:50 Error: [Error] Call to a member function success() on null
Request URL: /acl/users/add
Stack Trace:
#0 [internal function]: UsersController->add()
#1 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#2 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-02-15 13:23:05 Error: [Error] Call to a member function success() on null
Request URL: /acl/users/add
Stack Trace:
#0 [internal function]: UsersController->add()
#1 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#2 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#4 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-02-15 13:24:50 Error: [Error] Call to a member function success() on null
Request URL: /acl/posts/add
Stack Trace:
#0 [internal function]: PostsController->add()
#1 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#2 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest))
#4 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-02-15 13:24:58 Error: [Error] Call to a member function success() on null
Request URL: /acl/posts/delete/1
Stack Trace:
#0 [internal function]: PostsController->delete('1')
#1 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#2 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest))
#4 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
2021-02-15 13:25:51 Error: [Error] Call to a member function success() on null
Request URL: /acl/posts/add
Stack Trace:
#0 [internal function]: PostsController->add()
#1 C:\wamp64\www\acl\lib\Cake\Controller\Controller.php(499): ReflectionMethod->invokeArgs(Object(PostsController), Array)
#2 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#3 C:\wamp64\www\acl\lib\Cake\Routing\Dispatcher.php(167): Dispatcher->_invoke(Object(PostsController), Object(CakeRequest))
#4 C:\wamp64\www\acl\app\webroot\index.php(117): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#5 {main}
